gulpfile.js

```diff
@@ -88,7 +88,7 @@ gulp.task('test.e2e', ['test.compile'], function(done) {
 
   // run node with a shell so we can wildcard all the .ts files
   var cmd = 'node ../lib/main.js --translateBuiltins --basePath=. --destination=. ' +
-      '*.ts angular2/src/facade/lang.d.ts';
+      '*.ts angular2/src/facade/lang.d.ts typings/es6-promise/es6-promise.d.ts';
```

Comment:
wait, we should not need to add anything, there is already a tsd.json in the project, just need to add es6-promise to it.